### PR TITLE
NAS-109754 / 21.06 / [Charts/Apps] Lists-in-Lists broken on latest build

### PR DESCRIPTION
**Pull request by AlexKarpov98**

### CODE REVIEW

File: src/app/pages/common/entity/entity-form/components/form-dict/form-dict.component.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<div id="{{config.name}}" fxLayout="column" class="dynamic-field mat-dict" [formGroup]="group" [ngClass]="fieldShow" [class.has-tooltip]="config.tooltip" *ngIf="!config['isHidden']">
```
Comment (undsoft) [inline src/app/pages/common/entity/entity-form/components/form-dict/form-dict.component.html:1]: [id]="config.name"

File: src/app/pages/common/entity/entity-form/components/form-dict/form-dict.component.html [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+  <label class="label input-text">{{config.label | translate}}</label>
+  <ng-container *ngFor="let field of config.subFields;" >
+    <div *ngIf="!field.isHidden" fxFlex="calc(100% - 100px)"
+      [ngClass]="{'form-inline': field.class == 'inline', 'form-line': field.class !== 'inline'}"
```
Comment (undsoft) [inline src/app/pages/common/entity/entity-form/components/form-dict/form-dict.component.html:5]: How about:
[class.form-inline]="field.class === 'inline'",
[class.form-line]="field.class !== 'inline'

File: src/app/pages/common/entity/entity-form/components/form-dict/form-dict.component.ts [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+import { FieldConfig } from '../../models/field-config.interface';
+import { Field } from '../../models/field.interface';
+import { EntityFormService } from '../../services/entity-form.service';
+import {FieldRelationService} from '../../services/field-relation.service';
+
+@Component({
+  selector: 'entity-form-dict',
+  templateUrl: './form-dict.component.html',
+  styleUrls: ['./form-dict.component.css', '../dynamic-field/dynamic-field.css'],
```
Comment (undsoft) [inline src/app/pages/common/entity/entity-form/components/form-dict/form-dict.component.ts:13]: First css is empty and can be removed.

**Comment by AlexKarpov98** [comment]
@undsoft Fixed all issues you mentioned above.

**Review by sonicaj — COMMENTED** [review]
I have following concerns:

1) I am not able to see CMD specified for an installed app launched with `Launch Docker Image` wizard.

Screenshot 2021-05-03 at 7 12 06 AM

2) I see this logged each time i open a plex app for editing

Screenshot 2021-05-03 at 7 11 06 AM

( I have DM you on slack for server credentials where this can be found )

**Comment by AlexKarpov98** [comment]
> I have following concerns:
> 
> 1. I am not able to see CMD specified for an installed app launched with `Launch Docker Image` wizard.
> 
> Screenshot 2021-05-03 at 7 12 06 AM
> 
> 1. I see this logged each time i open a plex app for editing
> 
> Screenshot 2021-05-03 at 7 11 06 AM
> 
> ( I have DM you on slack for server credentials where this can be found )

@sonicaj I can't reproduce 1st issue.
Can you please check once again?

**Comment by sonicaj** [comment]
@AlexKarpov98 please see `cmd-app` on that machine, it has CMD specified in it's config but it's not being rendered in the UI

**Comment by AlexKarpov98** [comment]
@sonicaj Fixed above issues.

**Review by sonicaj — APPROVED** [review]
Thanks! Although just to be sure, is this desired behavior ? ( this for `plex` app on the same system )

Screenshot 2021-05-05 at 7 17 18 AM

**Review by undsoft — APPROVED** [review]
Thanks for the fixes.

**Pull request merged**